src/app/problemList/codeEditor/langSelection.js

```diff
@@ -33,15 +33,15 @@ class LangSelection extends React.Component {
 
     handleChoice(newLang) {
         this.setState({
-            anchorEl: null,
+            anchorEl: undefined,
         })
         this.props.handleChange(newLang)
```

Comment:
What does this mean ?

Reply:
I asking about `this.props.handleChange(newLang)`

Reply:
See this : 

https://github.com/vnma0/hestia/blob/bd18953dbd34ba0cc0807fa946a882e20a840bb3/src/app/problemList/codeEditor/codeBox.js#L56

Reply:
At least add some comments

Reply:
Psh, as of #98, these things would have been rewritten already.